Can't build fonts on Debian 11 Bullseye

Dear dev,
when I try to build the font, I get this error:
```
$ make
Generating	build/EBGaramond08-Regular.otf
make: python: No such file or directory
make: *** [Makefile:53: build/EBGaramond08-Regular.otf] Error 127
```

All dependencies are installed:
```
$ apt policy python3-fontforge ttfautohint 
python3-fontforge:
  Installed: 1:20201107~dfsg-4
  Candidate: 1:20201107~dfsg-4
  Version table:
 *** 1:20201107~dfsg-4 500
        500 http://deb.debian.org/debian bullseye/main amd64 Packages
        100 /var/lib/dpkg/status
ttfautohint:
  Installed: 1.8.3-2+b1
  Candidate: 1.8.3-2+b1
  Version table:
 *** 1.8.3-2+b1 500
        500 http://deb.debian.org/debian bullseye/main amd64 Packages
        100 /var/lib/dpkg/status
```

Maybe the above libraries are old?

--- a/Makefile
+++ b/Makefile
@@ -19,7 +19,7 @@ DIST=$(NAME)-$(VERSION)-complete
 #Call script through fontforge, not python. https://github.com/fontforge/fontforge/issues/528
 FF=fontforge
 #Return to python because we don’t scale the font any longer.
-PYTHON?=python
+PYTHON?=python3
 SCRIPT=tools/makefont.py
 
 #SIZES=08 12

--- a/Makefile
+++ b/Makefile
@@ -19,7 +19,11 @@ DIST=$(NAME)-$(VERSION)-complete
 #Call script through fontforge, not python. https://github.com/fontforge/fontforge/issues/528
 FF=fontforge
 #Return to python because we don’t scale the font any longer.
-PYTHON?=python3
+ifeq ($(OS),Windows_NT)
+    PYTHON?=ffpython
+else
+    PYTHON?=python3
+endif
 SCRIPT=tools/makefont.py
 
 #SIZES=08 12

